pkg/apis/core/v1alpha1/cmdimage_types.go

```diff
@@ -69,6 +69,16 @@ type CmdImageSpec struct {
 	// +tilt:local-path=true
 	Dir string `json:"dir,omitempty" protobuf:"bytes,2,opt,name=dir"`
 
+	// Additional environment variables for the build process.
+	//
+	// Expressed as a C-style array of strings of the form ["KEY1=VALUE1", "KEY2=VALUE2", ...].
+	//
+	// Environment variables are layered on top of the environment variables
+	// that Tilt runs with.
+	//
+	// +optional
+	Env []string `json:"env,omitempty" protobuf:"bytes,10,rep,name=env"`
```

Comment:
Do I need to run any code generation for this new argument? I don't think I updated the docs for this new field.

Reply:
The CI check caught some missing code gen and pointed me at the right command to run, I think.